### ISSUE TITLE
Accept Zowe JWT token for authentication using Zowe Security Client library

### ISSUE THREAD

**Issue by plavjanik** [zowe/sample-spring-boot-api-service#18]
- Follow the same request and response format as API Catalog
- https://zowe.org/docs-site/latest/extend/extend-apiml/api-mediation-security.html#security-service-client-library

**Comment by plavjanik** [comment]
Hi @JirkaAichler, it makes sense for us to work on it together or any two people who know both the sample and the security client.

**Comment by JirkaAichler** [comment]
I agree. I can talk to Petr Galik whether I'll be given some time for it in the next PI.

**Comment by plavjanik** [comment]
Duplicate of #83
